include/gqf.h

```diff
@@ -109,7 +109,7 @@ extern "C" {
 
 	void qf_destroy(QF *qf);
 
-	void qf_copy(QF *dest, QF *src);
+	void qf_copy(QF *dest, const QF *src);
```

Comment:
@shokrof how do you feel about making `src` into a `const QF *`? This makes my life easier (because of mutability and ownership rules in Rust), and should still be valid wherever `qf_copy` was called before, right?